src/db/dbQuery.ts

```diff
@@ -0,0 +1,39 @@
+import { database } from "./dbConfig";
+
+
+export const getAllUnsentEmailData =(email:boolean = true, email_sent:boolean = false, seen:boolean = false) => {
+    try {
+        return new Promise((resolve, reject) => {
+            database.query(`SELECT n.id, n.user_id, u.username, u.email, el.event_id
```

Comment:
In order to give more information about the events, you will need to fetch username and names of the related things.